ci/jenkins/test.sh

```diff
@@ -35,7 +35,8 @@ IMAGE_PULL_POLICY="Always"
 PROXY_ALL=false
 DEFAULT_IP_MODE="ipv4"
 IP_MODE=""
-K8S_VERSION="1.28.2-00"
+#K8S_VERSION="1.30.1-1.1"
+K8S_VERSION="1.28.9-2.1"
```

Comment:
The version in this line and in PR message should be updated to 1.30?

Reply:
This is debug code. Please ignore this.

Reply:
Fixed.